### PR TITLE
Generated the V1 of our database schema using flyway.

### LINKED ISSUE #11: Add Flyway migration.
Dependencies are already in pom.xml.

### PR DESCRIPTION
**Pull request by Win-ther**
### Related Issue(s)
Closes #11 

### Proposed Changes
- Generated V1 of our database using flyway.

### Description
By having flyway migration in our project, we are checking every upstart if the schema in our db is correct and if it does not exist, it will be created.

### CODE REVIEW

File: src/main/resources/db/migration/V1__.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+CREATE TABLE user
+(
+    id         BIGINT AUTO_INCREMENT NOT NULL,
+    user_name  VARCHAR(255)          NULL,
+    first_name VARCHAR(255)          NULL,
+    last_name  VARCHAR(255)          NULL,
+    email      VARCHAR(255)          NULL,
+    CONSTRAINT pk_user PRIMARY KEY (id)
+);
```
Comment (Frlundsten) [inline src/main/resources/db/migration/V1__.sql]: There is no column for the image.

**Review by Frlundsten — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V1__.sql (hunk 1 above)
Comment (Win-ther) [reply src/main/resources/db/migration/V1__.sql]: Oops missed that completely, good catch :)

**Review by Frlundsten — APPROVED** [review]
Nice 👍

**Review by marcusju23 — APPROVED** [review]
Good stuff! 🙌

**Review by altmannk — APPROVED** [review]
Great :)

**Pull request merged**